src/stores/data.js

```diff
@@ -93,12 +93,32 @@ async function getDatasetsAvgs3Rounds(api,roundNumber) {
         ],
         "group": [
           "nft_addr",
-          "ve_allocated",
-          "volume",
-          "apy"
-        ],
-        "sort": {
-            "volume": -1
+        ]
+      })
+    });
+  } catch (error) {
+    console.log(error);
+    return [];
+  }
+  let data = await res.json();
+  return data;
+}
+
+async function getDatasetsRoundValues(api,roundNumber) {
+  let res;
+  try {
+    res = await fetch(api, {
+      method: "POST",
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        "query":{
+          "round": {
+            "$in": getRoundsDatafarm(roundNumber, 5)
+          },
+          "nft_addr":"0x9723488dc1524849a82917a61a38bbe24a8219c1"
```

Comment:
Why are you requesting for this specific address?

Reply:
For testing purpose